lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,22 +24,33 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
```

Comment:
Sure. Given that this would basically be a complete rewrite, it's fine to not make these changes, but do try to read around the extra methods you have available to you, because it should hopefully lead to cleaner code.